### DISCUSSION TITLE
The x86 version of Cef.core.rutime.dll would load failed when dbghelp.dll exists in the build output folder.

### DISCUSSION THREAD

**Discussion by LuxWang** [cefsharp/CefSharp#3517] [General]
This issue is related to https://github.com/cefsharp/CefSharp/discussions/3478.
However we could find an easy way to reproduce it, so we created another discussion.

The way to reproduce: (Cefsharp 87.1.132 and 89.0.170 have the same issue)
1) download the CefSharp.MinimalExample
2) Open CefSharp.MinimalExample.net472.sln in CefSharp.MinimalExample-master folder. (visual studio 2019 would open the solution file)
3) Build it in this solution configuration and platform (Debug\x86)
4) Go to the build output folder: "CefSharp.MinimalExample-master\CefSharp.MinimalExample.WinForms\bin.net472\x86\Debug\net472"
5) Copy the attached dbghelp.dll to the build output folder.
6) Go back to visual studio 2019 and start debugging.

It would throw the following exception:
-		$exception	StackTrace = "   at CefSharp.Cef.EnableHighDPISupport()\r\n   at CefSharp.MinimalExample.WinForms.Program.Main(String[] args) in D:\\CefSharp\\CefSharp.MinimalExample-master\\CefSharp.MinimalExample.WinForms\\Program.cs:line 24"	System.IO.FileLoadException
		StackTrace	"   at CefSharp.Cef.EnableHighDPISupport()\r\n   at CefSharp.MinimalExample.WinForms.Program.Main(String[] args) in D:\\CefSharp\\CefSharp.MinimalExample-master\\CefSharp.MinimalExample.WinForms\\Program.cs:line 24"	string
+		Data	{System.Collections.ListDictionaryInternal}	System.Collections.IDictionary {System.Collections.ListDictionaryInternal}
		FileName	"CefSharp.Core.Runtime.dll"	string
		FusionLog	""	string
		HResult	-2147024769	int
		HelpLink	null	string
+		InnerException	null	System.Exception
		Message	"A procedure imported by 'CefSharp.Core.Runtime.dll' could not be loaded."	string
		Source	"CefSharp.Core"	string
+		TargetSite	{Void EnableHighDPISupport()}	System.Reflection.MethodBase {System.Reflection.RuntimeMethodInfo}

The dbghelp.dll is used in another project in our solution, so it would exist in the build output folder.
However, we could not load cefsharp component when dbghelp.dll exists.

Do you have any idea to fix this issue? or maybe we should update dbghelp to some specified version?

Thank you so much for your help.
[dbghelp.zip](https://github.com/cefsharp/CefSharp/files/6358954/dbghelp.zip)

**Comment by amaitland** [comment]
> This issue is related to #3478.

In future please use your original discussion rather than opening a new one.

> Copy the attached dbghelp.dll to the build output folder.

For security reasons I won't be downloading this file.

> Do you have any idea to fix this issue? or maybe we should update dbghelp to some specified version?

dbghelp.dll Is arch specific, do you have the correct bitness? Try obtaining the latest version from the Windows 10 SDK as per https://docs.microsoft.com/en-us/windows/win32/debug/dbghelp-versions

**Comment by LunevNF** [comment]
Same problem: "A procedure imported by 'CefSharp.Core.Runtime.dll' could not be loaded.". AnyCPU, latest version of lib.

**Reply by LunevNF** [reply]
My solution:
1. Install via nuget cef.redist.x64 v103.0.12
2. Install via nuget cef.redist.x86 v103.0.12
3. Install via nuget CefSharp.Common v103.0.120
4. Install via nuget CefSharp.Wpf v103.0.120
---
Life is good.
